generators/client/templates/vue/package.json.ejs

```diff
@@ -27,14 +27,14 @@ limitations under the License.
   ],
   "dependencies": {
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
-    "@fortawesome/fontawesome-svg-core": "1.2.30",
-    "@fortawesome/free-solid-svg-icons": "5.14.0",
-    "@fortawesome/vue-fontawesome": "0.1.10",
-    "axios": "0.19.2",
-    "bootstrap": "4.5.0",
-    "bootstrap-vue": "2.14.0",
+    "@fortawesome/fontawesome-svg-core": "1.2.32",
```

Comment:
@atomfrede Can you migrate some of the dependencies to package.json file so the updates can be managed by dependabot automatically?